tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -55,6 +55,235 @@ def _emit_event(self, resource, curr_value, msg, instance):
             )
         )
 
+    def _sync_cluster_utilization_details(self, volumes):
```

Comment:
this is syncing vol and cluster utilization right, can we have the funtion name accordingly